### ISSUE TITLE
Localization file template update on crowdin

### ISSUE THREAD

**Issue by NorwayFun** [Exiv2/exiv2#2876]
Hello,

Wanted to translate exiv2 to Georgian. Found project page on crowdin, read instructions and emailed Leonardo for adding language.
Language was added right away, but I was told to contact project mantainers to update .pot file on crowdin, because it's too old, so here I am.

BR, Temuri

**Comment by NorwayFun** [comment]
P.S. 
There is no "download po file", nor upload in crowdin menu. Maybe there is something to enable for it?

**Comment by kmilos** [comment]
Hi @NorwayFun, I have now updated the exiv2.pot in https://github.com/Exiv2/exiv2/pull/2945

**Comment by NorwayFun** [comment]
Hello,
Thank you, but I cannot see it here https://crowdin.com/project/exiv2/activity-stream
Also, pot file upload functionality is still disabled.

**Comment by kmilos** [comment]
No idea how crowdin works, that page is not maintained by the Exiv2 project AFAIK.

You might want to reach out to the contact person there... @lbschenkel

**Comment by lbschenkel** [comment]
Basically the `exiv2.pot` file in the repo needs to be updated to reflect the source code, i.e., the strings needs to be extracted with `gettext` and the file committed. Once it's up-to-date, I can upload that to Crowdin so translators can translate those strings, and at some point the `.po` files with the translations can be downloaded and committed to the repo.

This is better automated, of course, but we can do it manually a few times until the workflow is understood by everyone.

**Comment by kmilos** [comment]
@lbschenkel As I mentioned already, [the template update has been done](https://github.com/Exiv2/exiv2/blob/0.28.x/po/exiv2.pot).

**Comment by lbschenkel** [comment]
> @lbschenkel As I mentioned already, [the template update has been done](https://github.com/Exiv2/exiv2/blob/0.28.x/po/exiv2.pot).

I was vacationing, so I'm not following very closely.

Now as a next step I can update the `pot` file in Crowdin, which will allow the (new) strings to be translated. I can do that myself but I'm wondering if somebody else wants to do that / see how it works, so we increase the bus factor to something slightly larger than 1?

Regarding the permissions, I'm more than happy to grant all the rights / relinquish ownership of the Crowdin project to the Exiv2 maintainers. I set this up myself but on request from @clanmills (see https://github.com/Exiv2/exiv2/issues/1510). The original intent was always to put the maintainers in control and to (semi-)automate the workflow.

**Comment by lbschenkel** [comment]
Uploaded `exiv2.pot` from `0.28.x` branch to Crowdin. I wonder why we're only doing this for this branch, and not `master` too?

**Comment by lbschenkel** [comment]
@NorwayFun: have fun translating

At some point in the future we should sync the translations back, meaning: I will download the `.po` files and make a PR against the project. I will let the maintainers decide when they want to do this (based on time or milestone), and I can put a reminder and submit a PR when that happens.

**Comment by NorwayFun** [comment]
@lbschenkel  Thanks much!
Just... there are 3 exiv2 pot files. which one should I translate, please?
Also, I cannot see "download/upload" option on crowdin.
Should look like this:
![image](https://github.com/Exiv2/exiv2/assets/72336380/5fe553f5-ca98-4bdd-9da2-e979cf65fe8a)

but it does look like this:

![image](https://github.com/Exiv2/exiv2/assets/72336380/e0f626ab-5fed-4026-9e6f-9c00fe54300a)

Temuri

**Comment by lbschenkel** [comment]
@NorwayFun: you don't need to worry about files at all, you go to Dasbhboard and pick the language, and you'll reach somewhere like: https://crowdin.com/editor/exiv2/all/en-bs?view=comfortable

Regarding upload/download, you don't need to do any of that, that's for the maintainers (or me) to do. Translators just use the web UI to translate.

**Comment by NorwayFun** [comment]
Yes, you're right, it's just me having almost half a million entries in my poedit suggestion list, so I'd prefer to dl it, translate on my pc and upload it after.

Temuri

**Comment by lbschenkel** [comment]
> Yes, you're right, it's just me having almost half a million entries in my poedit suggestion list, so I'd prefer to dl it, translate on my pc and upload it after.

OK, in that case I have changed the project settings and you should have seen a new option in the translation UI that allows you to download the PO file, work offline, and upload the translations.

**Comment by NorwayFun** [comment]
Awesome! Thanks much!

**Comment by kmilos** [comment]
> At some point in the future we should sync the translations back, meaning: I will download the `.po` files and make a PR against the project. I will let the maintainers decide when they want to do this (based on time or milestone), and I can put a reminder and submit a PR when that happens.

Great, thanks. There is a tentative 0.28.3 [milestone](https://github.com/Exiv2/exiv2/milestones) AFAICT, but the date is not set in stone...

**Comment by lbschenkel** [comment]
> @lbschenkel Please commit any translation updates you might have for the 0.28.x branch, looks like we might be preparing a point release soon...

Please check https://github.com/Exiv2/exiv2/pull/2986. I am assuming that you were referring to the `0.28.x` branch.
